cnf-tests/testsuites/e2esuite/security/tuning.go

```diff
@@ -0,0 +1,88 @@
+package security
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	client "github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/client"
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/execute"
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/namespaces"
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/networks"
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/pods"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+const (
+	TestNamespace = "tuning-testing"
+	Sysctl        = "net.ipv4.conf.IFNAME.send_redirects"
+)
+
+var _ = Describe("tuningcni", func() {
+	apiclient := client.New("")
+
+	execute.BeforeAll(func() {
+		err := namespaces.Create(TestNamespace, apiclient)
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	AfterEach(func() {
+		err := namespaces.CleanPods(TestNamespace, apiclient)
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	Context("tuningcni over macvlan", func() {
+
+		It("should be able to create pod with sysctls over macvlan",
+			func() {
+				nadName := "tuning-nad"
+				nad, err := networks.NewNetworkAttachmentDefinitionBuilder(TestNamespace, nadName).WithMacVlan("10.10.0.1").WithTuning(map[string]string{Sysctl: "1"}).Build()
+				Expect(err).ToNot(HaveOccurred())
+				err = client.Client.Create(context.Background(), nad)
+				Expect(err).ToNot(HaveOccurred())
+				podDefinition := pods.DefineWithNetworks(TestNamespace, []string{fmt.Sprintf("%s/%s", TestNamespace, nadName)})
+				pod, err := client.Client.Pods(TestNamespace).Create(context.Background(), podDefinition, metav1.CreateOptions{})
+				Expect(err).ToNot(HaveOccurred())
+				Expect(err).ToNot(HaveOccurred())
+
+				err = pods.WaitForPhase(client.Client, pod, corev1.PodRunning, 1*time.Minute)
+				Expect(err).ToNot(HaveOccurred())
```

Comment:
should you check the status of the sysctl inside the container when it's running?

Reply:
basic tests for this are already done in origin
but added this anyway